From 9aa09cffed8c656ad3bc11fe343b663da1c834c7 Mon Sep 17 00:00:00 2001
From: Andreas Enge <andreas.enge@inria.fr>
Date: Fri, 8 Jun 2018 16:54:33 +0200
Subject: [PATCH 12/29] m4: Disable -Werror completely.

* m4/mpc.m4: Do not enable -Werror, not even for the user enge. With gcc-7.3
  at least, limits.h uses the gcc extension #include_next, which triggers a
  warning.
---
 m4/mpc.m4 | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/m4/mpc.m4 b/m4/mpc.m4
index 7ed4bd3..f3c7554 100644
--- a/m4/mpc.m4
+++ b/m4/mpc.m4
@@ -1,6 +1,6 @@
 # mpc.m4
 #
-# Copyright (C) 2008, 2009, 2010, 2011, 2012, 2014 INRIA
+# Copyright (C) 2008, 2009, 2010, 2011, 2012, 2014, 2018 INRIA
 #
 # This file is part of GNU MPC.
 #
@@ -91,10 +91,6 @@ AC_DEFUN([MPC_C_CHECK_WARNINGCFLAGS], [
   AC_REQUIRE([AC_PROG_GREP])
   if echo $VERSION | grep -c dev >/dev/null 2>&1 ; then
     if test "x$GCC" = "xyes" -a "x$compiler" != "xicc"; then
-      # enable -Werror for myself (Andreas Enge)
-      if test "x$USER" = "xenge"; then
-         MPC_C_CHECK_FLAG(-Werror)
-      fi
       MPC_C_CHECK_FLAG(-g)
       MPC_C_CHECK_FLAG(-std=c99)
       MPC_C_CHECK_FLAG(-Wno-long-long)
-- 
2.25.0

